masterlist.yaml

```
@@ -837,6 +837,22 @@ plugins:
         display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
         condition: 'not active("MasterIndexRedux.esp")'
 
+  - name: 'Main_Quest_Overhaul.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/46913/' ]
+    inc:
+      - 'Ashlander Quests.esp'
+      - 'Consequences for Looting the Andrano Tomb.esp'
+      - 'Exterminate all the Brutes.esp'
+      - 'Hortator.esp'
+      - 'Imperialist Nerevarine.esp'
+      - 'Joinable Sixth House - So to Speak.esp'
+      - 'Living with Caius.esp'
+      - 'Nationalist Nerevarine.esp'
+      - 'Past Life Regressions.esp'
+      - 'Refuse the Main Quest.esp'
+      - 'Sixth House Smugglers.esp'
+      - 'Vampire Nerevarine.esp'
```

`Vampire Nerevarine.esp` appears to actually be `Vampire_Nerevarine.ESP`. Also, it'd be preferable for capitalization to match between the mods & the masterlist:

```suggestion
      - 'Ashlander Quests.ESP'
      - 'Consequences for Looting the Andrano Tomb.ESP'
      - 'Exterminate all the Brutes.ESP'
      - 'Hortator.ESP'
      - 'Imperialist Nerevarine.ESP'
      - 'Joinable Sixth House - So to Speak.ESP'
      - 'Living With Caius.ESP'
      - 'Nationalist Nerevarine.ESP'
      - 'Past Life Regressions.ESP'
      - 'Refuse the Main Quest.ESP'
      - 'Sixth House Smugglers.ESP'
      - 'Vampire_Nerevarine.ESP'
```